teuthology-pull-requests/config/definitions/teuthology-pull-requests.yml

```diff
@@ -70,3 +70,6 @@
           !include-raw:
             - ../../../scripts/build_utils.sh
             - ../../build/build
+
+    publishers:
+      - github-notifier
```

Comment:
It may be that I'm doing it wrong, but after I enabled "Require status checks..." [here](https://github.com/ceph/teuthology/settings/branches/master), the status wasn't being updated.

Now I see that there are two actual status checks (I swear there was only the one before); perhaps if that were checked we wouldn't need this second plugin?